query/src/main/scala/filodb/query/ProtoConverters.scala

```diff
@@ -3,13 +3,10 @@ package filodb.query
 
 import java.util.concurrent.TimeoutException
 import java.util.concurrent.atomic.{AtomicInteger, AtomicLong}
-
 import scala.collection.JavaConverters._
```

Comment:
These empty lines are needed for style checks, add them back.

Reply:
sorry, this is just a typo